helm/alfresco-content-services/templates/Rule_04-network-policy-repository.yaml

```diff
@@ -70,11 +72,13 @@ spec:
       - podSelector:
           matchLabels:
              app: {{ template "content-services.shortname" . }}-repository
-      
+
+      {{ if index .Values "alfresco-search" "enabled" }}
```

Comment:
some of these newly added "if"s start with 
{{- if 

others
{{ if

find out why or be consistent across the file